Sources/HTTP/HTTPBodyDecoder.swift

```diff
@@ -89,7 +89,7 @@ extension JSONEncoder: HTTPBodyEncoder {
 
 extension JSONDecoder: HTTPBodyDecoder {
     /// See `HTTPBodyDecoder.decode(_:from:maxSize:on:)`
-    public func decode<D>(_ decodable: D.Type, from body: HTTPBody, maxSize: Int, on worker: Worker) throws -> EventLoopFuture<D> where D : Decodable {
+    public func decode<D>(_ decodable: D.Type, from body: HTTPBody, maxSize: Int = 65_536, on worker: Worker) throws -> EventLoopFuture<D> where D : Decodable {
```

Comment:
I think this would be better added as an extension to `HTTPBodyDecoder` so that all implementations get access to it.

Reply:
@tanner0101 Should i go ahead and add that or you wanna handle that?

Reply:
i can do it :)

Reply:
Kewl